### ISSUE TITLE
Browser does not reload with ts references

### ISSUE THREAD

**Issue by vitalikyarina** [angular-architects/module-federation-plugin#722]
### With what library do you have an issue?

native-federation

### Reproduction of the bug/regression with instructions

I have tsconfig with ts references like this
{
  "compilerOptions": {
    "paths": {
      "@core/*": ["src/app/core/*"],
      "@ng/ui/menu": ["../../shared/angular/components/menu"]
    }
  },
  "references": [
    { "path": "../../shared/angular/core/tsconfig.lib.json" },
    { "path": "../../shared/angular/components/tsconfig.lib.json" }
  ]
}
1. pnpm start
2. Change something in ../../shared/angular/components/menu/*
![image](https://github.com/user-attachments/assets/87381e8a-7947-460e-abe3-6f9c92dc8c8a)
4. Check changes in browser

### Expected behavior

When do point 2 the build works good, all changes is on dist directory but page doesn't reload, if i reload myself everything works
I thinks its must reload page if some changed in references libs

### Versions of Native/Module Federation, Angular, Node, Browser, and operating system

"@angular-architects/native-federation": "^19.0.2",
"@angular/core": "^19.0.5",
node: 20.16.0,
Google Chrome: 131.0.6778.205
Windows 11 23H2

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [x] No

**Comment by manfredsteyer** [comment]
You are right. The thing is, here, Native Federation sees the shared lib as something external (like a npm package) and does not reload automatically.

**Comment by vitalikyarina** [comment]
> You are right. The thing is, here, Native Federation sees the shared lib as something external (like a npm package) and does not reload automatically.

Any way to add libs which need to track?
